sidebars.js

```diff
@@ -152,7 +150,6 @@ module.exports = {
 					items: [
 						'customization/yoast-seo/api-filter-actions-deprecations',
 						'customization/yoast-seo/adding-custom-data-analysis',
-						'customization/yoast-seo/indexables-cli',
```

Comment:
Why was this entry completely removed instead of being moved?

Reply:
This is moved to the `features/wp-cli/indexables-cli` path.
That resulted in the wrongly named tmp file I think 😊